.github/workflows/aws-cleanup.yaml

```diff
@@ -0,0 +1,31 @@
+name: AWS cleanup
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch: {}
+
+jobs:
+  cleanup:
+    name: Cleanup AWS resources
+    environment: "cleanup AWS"
```

Comment:
```suggestion
    environment: "Integration tests"
```

Reply:
The [environment is an actiual thing in Github](https://docs.github.com/en/actions/reference/environments)